server/cli/launch.py

```diff
@@ -0,0 +1,96 @@
+import click
+import logging
+import webbrowser
+
+from os.path import splitext, basename, isfile
+
+
+@click.command()
+@click.argument('data', metavar='<data file>')
+@click.option('--layout', '-l', type=click.Choice(['umap', 'tsne']), default='umap', show_default=True,
+              help='Method for layout.')
+@click.option('--diffexp', '-d', type=click.Choice(['ttest']), default='ttest', show_default=True,
+              help='Method for differential expression.')
+@click.option('--title', '-t', help='Title to display (if omitted will use file name).', metavar='')
+@click.option('--verbose', '-v', is_flag=True, default=False, show_default=True,
+              help='Provide verbose output, including warnings and all server requests.')
+@click.option('--debug', '-d', is_flag=True, default=False, show_default=True,
+              help='Run in debug mode.')
+@click.option('--open', '-o', 'open_browser', is_flag=True, default=False, show_default=True,
+              help='Open the web browser after launch.')
+@click.option('--port', '-p', help="Port to run server on.", metavar='', default=5005, show_default=True)
+@click.option('--listen-all', is_flag=True, default=False, show_default=True,
+              help='Bind to all interfaces (this makes the server accessible beyond this computer).')
+@click.option('--max-category-items', default=100, metavar='', show_default=True,
+              help='Limits the number of categorical annotation items displayed.')
+def launch(data, layout, diffexp, title, verbose, debug, open_browser, port, listen_all, max_category_items):
+    """
+    Launch the cellxgene data viewer.
+    This web app lets you explore single-cell expression data.
+    Data must be in a format that cellxgene expects, read the
+    "getting started" guide.
+
+    Examples:
+
+    > cellxgene launch example_dataset/pbmc3k.h5ad --title pbmc3k
+
+    > cellxgene launch <your data file> --title <your title>
+    """
+    # Startup message
+    click.echo('[cellxgene] Starting the CLI...')
+
+    # Import Flask app
+    from ..app.app import app
```

Comment:
We're doing relative apps down, but not up. Can you make this:
```suggestion
    from server.app.app import app
```

Reply:
addressed